src/mixins/callInfoMixin.js

```diff
@@ -0,0 +1,38 @@
+export default {
+  data: () => ({
+    now: Date.now(),
+  }),
+
+  mounted() {
+    setInterval(() => {
+      this.now = Date.now();
+    }, 1000);
+  },
```

Comment:
wrap in function and call optionally only if needed. clear interval after destroyed()